Soccar [Game-Server]/src/nl/soccar/gameserver/model/session/SessionWrapper.java

```diff
@@ -11,25 +13,48 @@
     private final RoomWrapper room;
     private final GameWrapper game;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)